refl1d/reflectivity.py

```diff
@@ -238,33 +248,36 @@ def calculate_u1_u3(H, rhoM, thetaM, Aguide):
 
     return sld_b, u1, u3
 
+
 def calculate_u1_u3_py(H, rhoM, thetaM, Aguide):
     rotate_M = True
 
-    EPS = np.finfo('f').tiny # not 1e-20 # epsilon offset for divisions.
+    EPS = np.finfo('f').tiny  # not 1e-20 # epsilon offset for divisions.
     thetaM = radians(thetaM)
     phiH = radians(Aguide - BASE_GUIDE_ANGLE)
-    thetaH = np.pi/2.0 # by convention, H is in y-z plane so theta = pi/2
+    thetaH = np.pi/2.0  # by convention, H is in y-z plane so theta = pi/2
 
     sld_h = B2SLD * H
     sld_m_x = rhoM * np.cos(thetaM)
     sld_m_y = rhoM * np.sin(thetaM)
-    sld_m_z = 0.0 # by Maxwell's equations, H_demag = mz so we'll just cancel it here
+    sld_m_z = 0.0  # by Maxwell's equations, H_demag = mz so we'll just cancel it here
     # The purpose of AGUIDE is to rotate the z-axis of the sample coordinate
     # system so that it is aligned with the quantization axis z, defined to be
     # the direction of the magnetic field outside the sample.
     if rotate_M:
         # rotate the M vector instead of the transfer matrix!
         # First, rotate the M vector about the x axis:
-        new_my = sld_m_z * sin(radians(Aguide)) + sld_m_y * cos(radians(Aguide))
-        new_mz = sld_m_z * cos(radians(Aguide)) - sld_m_y * sin(radians(Aguide))
+        new_my = sld_m_z * sin(radians(Aguide)) + \
+                               sld_m_y * cos(radians(Aguide))
+        new_mz = sld_m_z * cos(radians(Aguide)) - \
+                               sld_m_y * sin(radians(Aguide))
```

Comment:
PEP8 discourages the use of backslash for line extension. Use parens instead.  And break before the operator.
```
new_my = (sld_m_z * sin(radians(Aguide))
    + sld_m_y * cos(radians(Aguide))
```
I'm guessing *Black* would do:
```
new_my = (
    sld_m_z * sin(radians(Aguide))
    + sld_m_y * cos(radians(Aguide)
)
```
I prefer fewer vertical lines, but the other version is defensible.

Reply:
I think VS code did that to me.  I don't know why it's not following PEP8